reana_workflow_controller/rest/utils.py

```diff
@@ -600,35 +603,33 @@ def get_workspace_diff(workflow_a, workflow_b, brief=False, context_lines=5):
     """
     workspace_a = workflow_a.workspace_path
     workspace_b = workflow_b.workspace_path
-    reana_fs = fs.open_fs(app.config["SHARED_VOLUME_PATH"])
-    if reana_fs.exists(workspace_a) and reana_fs.exists(workspace_b):
+    # reana_fs = fs.open_fs(app.config["SHARED_VOLUME_PATH"])
+    if os.path.exists(workspace_a) and os.path.exists(workspace_b):
         diff_command = [
             "diff",
             "--unified={}".format(context_lines),
             "-r",
-            reana_fs.getospath(workspace_a),
-            reana_fs.getospath(workspace_b),
+            bytes(workspace_a, "utf-8"),
+            bytes(workspace_b, "utf-8"),
```

Comment:
I think `bytes()` is not really needed here, since we use directories as strings anyway

Reply:
Yes, correct